infrastructure/terraform/modules/templates-api/variables.tf

```diff
@@ -55,8 +55,14 @@ variable "log_retention_in_days" {
 
 variable "cognito_config" {
   type = object({
-    USER_POOL_ID: string,
-    USER_POOL_CLIENT_ID: string
+    USER_POOL_ID : string,
+    USER_POOL_CLIENT_ID : string
   })
   description = "Cognito config"
 }
+
+variable "enable_sourcemaps" {
```

Comment:
I'm not sure we need a switch for this, not sure we would even switch it off.

Reply:
I was a bit confused about how to get it working properly - (I could only get it working with inline source maps, resulting in enormous files rather than a small file and a bigger source map) - so I thought it'd be good to have the option to turn it off. But I figured it out now, so have removed the flag.